### PR TITLE
Updates egress ip firewall note in live migration docs

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.15, 4.16

Issue:
https://issues.redhat.com/browse/OCPBUGS-42497

Link to docs preview:
https://85414--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-ovn-kubernetes-live-migration-about.adoc [hunk 1]
```diff
@@ -65,7 +65,7 @@ Before using the limited live migration method to the OVN-Kubernetes network plu
 
 * Egress router pods block the limited live migration process. They must be removed before beginning the limited live migration process.
 
-* During the limited live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the limited live migration process has finished.
+* During the migration, when the cluster is running with both OVN-Kubernetes and OpenShift SDN, multicast and egress IP are temporarily disabled for both CNIs. Egress firewalls remains functional.
```
Comment (mburke5678) [inline modules/nw-ovn-kubernetes-live-migration-about.adoc:68]: This is likely my ignorance, but I read _multicast and egress IP_ as _multicast IP and egress IP_, Would a typical user make that mistake? 
Similarly, in the original it uses _egress IP **addresses**_; does the rewrite need _addresses_ to be clear?

**Comment by mburke5678** [comment]
@stevsmit A couple of comments, likely rooted in my own ignorance of the subject matter. Proceed as you see fit.

File: modules/nw-ovn-kubernetes-live-migration-about.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/nw-ovn-kubernetes-live-migration-about.adoc:68]: fixed. either would've been ok IMO

**Pull request merged**